src/main/java/com/suse/salt/netapi/calls/modules/Pkg.java

```diff
@@ -259,6 +259,27 @@ private Pkg() { }
                 Optional.of(kwargs), new TypeToken<Map<String, Info>>(){});
     }
 
+    /**
+     * Call 'pkg.info_installed' API.
+     *
+     * @param attributes list of attributes that should be included in the result
+     * @param reportErrors if true will return an error message instead of corrupted text
+     * @param allVersions if true will return all installed package versions, not only latest
+     * @param packages optional give package names, otherwise return info about all packages
+     * @return the call
+     */
+    public static LocalCall<Map<String, Xor<Info, List<Info>>>> infoInstalled(List<String> attributes,
```

Comment:
like discussed i would remove the `allVersions` parameter here and pass always `true` since the return type only makes sense for `true` in case you would pass false you might as well use the old function with the simpler return type.